### ISSUE TITLE
Timecop doesn't provide #setDate()

### ISSUE THREAD

**Issue by searls** [jamesarosen/Timecop.js#11]
So after installing timecop and traveling, I have some code like this:

``` javascript
var inThirtyDays = new Date();
inThirtyDays.setDate(new Date().getDate() + 30);
```

Which, normally, would roll the date forward 30 days. However, if Date has been timecop'd, I instead get:

```
Result of expression 'inThirtyDays.setDate' [undefined] is not a function
```

**Comment by jamesarosen** [comment]
We probably need to make setters for [all 19 getters](https://github.com/jamesarosen/Timecop.js/blob/master/lib/MockDate.js#L10). Perhaps something like

``` javascript
var delegatedAspects = [
    'Date', 'Day', 'FullYear', 'Hours', 'Milliseconds', 'Minutes', 'Month',
    'Seconds', 'Time', 'TimezoneOffset', 'UTCDate', 'UTCDay', 
    'UTCFullYear', 'UTCHours', 'UTCMilliseconds', 'UTCMinutes',
    'UTCMonth', 'UTCSeconds', 'Year'
  ],
  delegatedActions = [ 'get', 'set' ],
  defineDelegate = function(method) {
    Timecop.MockDate.prototype[method] = function() {
      this._underlyingDate[method].apply(this, arguments);
    };
  };

for (var i = 0; i < delegatedActions.length; i++) {
  for (var j = 0; j < delegatedAspects.length; j++) {
    defineDelegate(delegatedActions[i] + delegatedAspects[j]);
  }
}
```

**Comment by jamesarosen** [comment]
Resolved in a8c3e9a176b669682cb210eb50660a2169e314fb
